docker-compose.yml

```diff
@@ -0,0 +1,6 @@
+app:
```

Comment:
Zas standardně - dev kontejner místo app

https://github.com/keboola/component-generator/blob/master/templates/php-component/docker-compose.yml

Reply:
Pardon my French, ale nie je toto uz trosku onania? :) Ved nech si to pomenuje dev/app/application/development/service/server - to je vsetko jedno. V readme predsa bude ako to mas pustit.

Reply:
No, třeba já mám `alias dcrb=docker-compose run --rm dev /bin/bash`

Reply:
Protože to skoro všude je dev... a je škoda, aby to na novém dev nebylo...

Reply:
No a ja mam alias `alias run=docker-compose run --rm cli` a ocakavam, ze vsade bude `cli` service s default command bash - tak ako na connection :)

Ten alias samozrejme nemam, ide mi skor o to neriesit blbosti a nerobit standard z niecoho, co je uplne obvious a imho netreba.

Reply:
No, ale jinde máme "app" jako to co nemá namountované volumes a je to na testy a k tomu pak dev, které je namountované má.... za mě je prostě lepší to držet a nedělat v tom zmatky...

Reply:
přejmenoval jsem to na dev - i když možná zbatečně :-)